src/common/translations/blank_before_percent.ts

```diff
@@ -0,0 +1,21 @@
+import { FrontendLocaleData } from "../../data/translation";
+
+// Logic based on https://en.wikipedia.org/wiki/Percent_sign#Form_and_spacing
+export const blankBeforePercent = (
+  localeOptions: FrontendLocaleData
+): string => {
+  switch (localeOptions.language) {
+    case "cz":
+    case "de":
+    case "en-gb":
+    case "fi":
+    case "fr":
+    case "nl":
```

Comment:
I followed the official rules as a guideline (at least for most, since Spanish seems to be a mixture as well). For Dutch I don't really care either way 🙈

Reply:
This is not a science context, we're a home automation app for consumers. We should follow the writing guidelines.

Reply:
Yeah, "science" might be a bad word in this context, but we are talking about showing values in a rather scientific way (not as part of a big text, but in a visual list in case of an Entities card where visual alignment/UX comes into play or Gauges cards). But I will remove the blank for Dutch again. Not my hill to die on...